### ISSUE TITLE
Input area covered by keyboard

### ISSUE THREAD

**Issue by KaiWW** [omz/Pythonista-Issues#595]
The input text field is covered by keyboard
![img_0037](https://user-images.githubusercontent.com/44955051/51787992-65d4a800-2170-11e9-8026-318c89e03dea.jpeg)

![img_0036](https://user-images.githubusercontent.com/44955051/51787984-3756cd00-2170-11e9-8d4f-75a3a496555c.png)

**Comment by wnMark** [comment]
@KaiWW is this reproduceable? Do you have this problem all the time? I think I had this problem one  time in first beta. But I was not able to reproduce.

**Comment by KaiWW** [comment]
@wnMark Yes It’s always there. I’m not using a beta version. I’m using an iPad 11 inch, it’s fine on my phone.

**Comment by KaiWW** [comment]
@wnMark just realised it happens when it’s in landscape mode.

**Comment by kylelambert101** [comment]
I'm having the same issue in my 11" iPad Pro. The on-screen keyboard always blocks the input bar in landscape. 
![img_0464](https://user-images.githubusercontent.com/19976093/52925449-7efbef00-32ff-11e9-90f6-eddcd0289f46.png)

**Comment by goatsweater** [comment]
I can reproduce the problem reliably on the 11" pro. Any time the app is in landscape (external keyboard or not), the special characters provided by the app always overlay the actual input area, so you can't see what you are typing.

Switching to portrait mode lets you see the input area.

**Comment by millere** [comment]
Hello, just wanted to chime in and say I’m also experiencing this issue on the 11” iPad Pro.

**Comment by JarvisBro** [comment]
I have the same problem with this. Hope there will be a fix soon.

**Comment by cdte92** [comment]
I have the same issue with gen 3 iPad Pro 12.9 inch with apple keyboard cover - so iPad is in landscape mode when using the keyboard.

**Comment by sDC2** [comment]
omz commented on this bug via twitter... goto settting\general\keyboard and disable 'shortcuts' as a temporary fix

**Comment by sDC2** [comment]
also this issue is the same as #575 in there the same solution is discussed
